src/odemis/acq/test/stream_test.py

```diff
@@ -38,8 +38,8 @@
 from odemis.driver import simcam
 from odemis.model import MD_POL_NONE, MD_POL_HORIZONTAL, MD_POL_VERTICAL, \
     MD_POL_POSDIAG, MD_POL_NEGDIAG, MD_POL_RHC, MD_POL_LHC, DataArrayShadow, TINT_FIT_TO_RGB
-from odemis.util import test, conversion, img, spectrum, find_closest
-from odemis.util.test import assert_array_not_equal
+from odemis.util import testing, conversion, img, spectrum, find_closest
+from odemis.util.testing import assert_array_not_equal
```

Comment:
can you update the code that uses `assert_array_not_equal` to use it as `testing.assert_array_not_equal` and then remove this line?